Source/WebCore/platform/graphics/cocoa/MediaPlaybackTargetContext.serialization.in

```diff
@@ -31,8 +33,22 @@ enum class WebCore::MediaPlaybackTargetContextMockState : uint8_t {
 enum class WebCore::MediaPlaybackTargetContextType : uint8_t {
     None,
     AVOutputContext,
-    SerializedAVOutputContext,
     Mock,
 };
 
+[Nested] struct WebCore::MediaPlaybackTargetContext::NonPlatformData {
+    String mockDeviceName;
+    WebCore::MediaPlaybackTargetContextMockState mockState;
+}
+
+[Nested] struct WebCore::MediaPlaybackTargetContext::PlatformData {
+    RetainPtr<AVOutputContext> outputContext;
+}
+
+class WebCore::MediaPlaybackTargetContext {
+    bool encodingRequiresPlatformData();
+    WebCore::MediaPlaybackTargetContextType type();
+    std::optional<WebCore::MediaPlaybackTargetContext::PlatformDataVariant> ipcData();
```

Comment:
I think @achristensen07 told me you have to use the full `std::variant<>` type explicitly here instead of the `PlatformDataVariant` alias for that our code generated knows about the type of the IPC data.